Gems/LyShine/Code/Source/UiCanvasComponent.cpp

```diff
@@ -294,6 +294,35 @@ namespace
 
         return canHandleEvents;
     }
+
+    bool IsAttachmentImageAssetCompatible(const AZ::Data::Asset<AZ::RPI::AttachmentImageAsset>& attachmentImageAsset)
```

Comment:
Declare this function static so that it only has internal linkage and is not accessible from other places.

Reply:
Actually this function is in an anonymous namespace so it shouldn't be accessible from other places.